tests/framework/Optimizers/GeneticAlgorithms/discrete/constrained/MinwoRepMultiObjectiveFeasibleFirst.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug" profile="jobs">
+  <TestInfo>
+    <name>raven/tests/framework/Optimizers/GeneticAlgorithms.NSGAII</name>

```

Comment:
For example, the name here need to be updated using the info from tests file. Please check my comments in tests file.

Reply:
Fixed.